src/stories/Introduction.stories.mdx

```diff
@@ -86,126 +89,43 @@ import StackAlt from './assets/stackalt.svg';
 
-<div className="link-list">
-  <a
-    className="link-item"
-    href="https://storybook.js.org/docs/react/addons/addon-types"
-    target="_blank"
-  >
-    <img src={Plugin} alt="plugin" />
-    <span>
-      <strong>Presets for popular tools</strong>
-      Easy setup for TypeScript, SCSS and more.
-    </span>
-  </a>
-  <a
-    className="link-item"
-    href="https://storybook.js.org/docs/react/configure/webpack"
-    target="_blank"
-  >
-    <img src={StackAlt} alt="Build" />
-    <span>
-      <strong>Build configuration</strong>
-      How to customize webpack and Babel
-    </span>
-  </a>
-  <a
-    className="link-item"
-    href="https://storybook.js.org/docs/react/configure/styling-and-css"
-    target="_blank"
-  >
-    <img src={Colors} alt="colors" />
-    <span>
-      <strong>Styling</strong>
-      How to load and configure CSS libraries
-    </span>
-  </a>
-  <a
-    className="link-item"
-    href="https://storybook.js.org/docs/react/get-started/setup#configure-storybook-for-your-stack"
-    target="_blank"
-  >
-    <img src={Flow} alt="flow" />
-    <span>
-      <strong>Data</strong>
-      Providers and mocking for data libraries
-    </span>
-  </a>
-</div>
+If you're new, start with the [**strating**](?path=/docs/starting--docs) guide, which will get you started with installation and library configuration.
 
-<div className="subheading">Learn</div>
+<div className="subheading">Links</div>
 
 <div className="link-list">
-  <a className="link-item" href="https://storybook.js.org/docs" target="_blank">
-    <img src={Repo} alt="repo" />
-    <span>
-      <strong>Storybook documentation</strong>
-      Configure, customize, and extend
-    </span>
-  </a>
-  <a className="link-item" href="https://storybook.js.org/tutorials/" target="_blank">
-    <img src={Direction} alt="direction" />
+  <a className="link-item" href="https://github.com/Popoyoko/popoyoko-ui" target="_blank">
+    <img src={github} alt="github" />
     <span>
-      <strong>In-depth guides</strong>
-      Best practices from leading teams
+      <strong>Github</strong>
+      View the source and add issues
     </span>
   </a>
-  <a className="link-item" href="https://github.com/storybookjs/storybook" target="_blank">
-    <img src={Code} alt="code" />
+  <a className="link-item" href="##" target="_blank">
+    <img src={npm} alt="npm" />
     <span>
-      <strong>GitHub project</strong>
-      View the source and add issues
+      <strong>Npm</strong>
+      Download package and use it
     </span>
   </a>
-  <a className="link-item" href="https://discord.gg/storybook" target="_blank">
-    <img src={Comments} alt="comments" />
+  <a className="link-item" href="##" target="_blank">
+    <img src={figma} alt="figma" />
     <span>
-      <strong>Discord chat</strong>
-      Chat with maintainers and the community
+      <strong>Figma</strong>
+      View the design, customize and make a token
```

Comment:
r
```suggestion
      View the design
```

Reply:
Resolved by this [commit](https://github.com/Popoyoko/popoyoko-ui/pull/201/commits/f8c858eefc99e35ca62f2ce404de3d87c1fceb6b).